packages/contracts/src/diamond/token/StakingShareForDiamond.sol

```diff
@@ -61,6 +61,7 @@ contract StakingShareForDiamond is ERC1155, ERC1155Burnable, ERC1155Pausable {
     /**
      * @dev constructor
      */
+    // TODO should this inherit from ERC1155UbiquityForDiamond?
```

Comment:
This `ForDiamond` naming convention makes me think that you are not using a dedicated token manager.